[Docs] Translate installation and 15_min (#629)
open-mmlab/mmengine: docs/en/get_started/15_minutes.md

# 15 minutes to get started with MMEngine

> Take training a ResNet-50 model on CIFAR-10 dataset as an example, we could build a complete, and configurable process for both training and validation in only 80 lines of code with `MMEgnine`.

Reviewer: ```suggestion
In this tutorial, we'll take training a ResNet-50 model on CIFAR-10 dataset as an example. We will build a complete and configurable pipeline for both training and validation in only 80 lines of code with `MMEgnine`.
```

The whole process includes the following steps:

1. [Build a Model](#build-a-model)
2. [Build a Dataset and DataLoader](#build-a-dataset-and-dataloader)
3. [Build a Evaluation Metrics](#build-a-evaluation-metrics)
4. [Build a Runner and Run the Task](#build-a-runner-and-run-the-task)

## Build a Model

> First, we need to build a **model**. In MMEngine, the model should inherit `BaseModel`. Besides accepting several parameters from the dataset, its `forward` method needs to accept an extra argument called `mode`:

Reviewer 2: ```suggestion
First, we need to build a **model**. In MMEngine, the model should inherit from `BaseModel`. Besides accepting several parameters from the dataset, its `forward` method needs to accept an extra argument called `mode`:
```

> First, we need to build a **model**. In MMEngine, the model should inherit from `BaseModel`. Besides accepting several parameters from the dataset, its `forward` method needs to accept an extra argument called `mode`:

Reviewer: ```suggestion
First, we need to build a **model**. In MMEngine, the model should inherit from `BaseModel`. Aside from parameters representing inputs from the dataset, its `forward` method needs to accept an extra argument called `mode`:
```

> - for training, the value of `mode` is "loss," and the `forward` method should return a   `dict` containing the key "loss".

Reviewer 2: ```suggestion
- for training, the value of `mode` is "loss," and the `forward` method should return a `dict` containing the key "loss".
```

                                      transforms.ToTensor(),
                                      transforms.Normalize(**norm_cfg)
                                  ])))

val_dataloader = DataLoader(batch_size=32,
                            shuffle=False,
                            dataset=torchvision.datasets.CIFAR10(
                                'data/cifar10',
                                train=False,
                                download=True,
                                transform=transforms.Compose([
                                    transforms.ToTensor(),
                                    transforms.Normalize(**norm_cfg)
                                ])))
```

## Build a Evaluation Metrics

To validate and test the model, we need to define a **Metric** called accuracy to evaluate the model. This metric needs inherit from `BaseMetric` and implements the `process` and `compute_metrics` methods where the `process` method accepts the output of the dataset and other outputs when `mode="predict"`. The output data at this scenario is a batch of data. After processing this batch of data, we save the information to `self.results` property.

> `compute_metrics` accepts a `results` parameter. The input of this parameter is all the information saved in `process` (In the case of a distributed environment, `results` are the information collected from all `process` in all the threads). Using these information to calculate and return a `dict` that holds the results of the evaluation metrics

Reviewer 2: ```suggestion
`compute_metrics` accepts a `results` parameter. The input `results` of `compute_metrics` is all the information saved in `process` (In the case of a distributed environment, `results` are the information collected from all `process` in all the processes). Using these information to calculate and return a `dict` that holds the results of the evaluation metrics
```

> `compute_metrics` accepts a `results` parameter. The input `results` of `compute_metrics` is all the information saved in `process` (In the case of a distributed environment, `results` are the information collected from all `process` in all the processes). Using these information to calculate and return a `dict` that holds the results of the evaluation metrics

Reviewer: ```suggestion
`compute_metrics` accepts a `results` parameter. The input `results` of `compute_metrics` is all the information saved in `process` (In the case of a distributed environment, `results` are the information collected from all `process` in all the processes). Use these information to calculate and return a `dict` that holds the results of the evaluation metrics
```
